components/next_steps_view/steps.test.tsx

```diff
@@ -15,6 +15,12 @@ describe('components/next_steps_view/steps', () => {
                 preferences: {
                     myPreferences: {},
                 },
+                users: {
+                    currentUserId: 'current_user_id',
+                    profiles: {
+                        current_user_id: {roles: 'system_role'},
+                    },
+                },
```

Comment:
Could we add a test for the steps for the case of non-admin ?

Reply:
I've added 2 small tests to make sure the filter steps logic is working based on a user's role.